bedrock/firefox/templates/firefox/dev-whatsnew.html

```diff
@@ -0,0 +1,112 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/developer" %}
+
+{% extends "/firefox/base-resp.html" %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_developer_firstrun' %}
+{% endblock %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Welcome to Firefox Developer Edition')}}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+{% block page_image %}{{ static('img/firefox/developer/page-image.png') }}{% endblock %}
+{% block page_og_url %}{{ url('firefox.developer') }}{% endblock %}
+
+{% block page_og_desc %}
+  {% if l10n_has_tag('meta_update_201503') %}
+    {{_('Built for those who build the Web. Introducing the only browser made for developers.')}}
+  {%- else -%}
+    {{ _('Introducing the only browser made for developers like you.') }}
+  {% endif %}
+{% endblock %}
+
+{% block body_id %}firefox-developer-whatsnew{% endblock %}
+{% block body_class %}blueprint{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}
+  <h2>{{ high_res_img('img/firefox/firstrun/dev/title.png', {'alt': _('Firefox Developer Edition'), 'width': '220', 'height': '84'}) }}</h2>
+{% endblock %}
+
+{% block content %}
+
+<main role="main">
+  <section class="intro container">
+    <header>
+      <h1>{{_('Congrats. You’re up to date!')}}</h1>
+      <p>{{_('Get to know the features that make it the most complete browser for building the Web.')}}</p>
+    </header>
+    <ul class="features">
+    {% if l10n_has_tag('spring_release') %}
+      <li class="feature{% if not LANG.startswith('en-') %} feature-centered{% endif %}">
+        <a href="https://developer.mozilla.org/docs/Tools/Performance" rel="external">
+          <img src="{{ static('img/firefox/firstrun/dev/feature-performancetools.png') }}" alt="{{ _('Screenshot') }}" class="screenshot">
+        </a>
+        <h2>{{_('Performance Tools')}}</h2>
+        <p>{{_('Examine your web content’s frame rate, memory consumption and JS profile in a consolidated timeline view to help easily identify and troubleshoot performance bottlenecks.')}}</p>
+        <a href="https://developer.mozilla.org/docs/Tools/Performance" rel="external" class="more">{{_('Learn more about Performance Tools')}}</a>
+        <div class="responsive-video-container">
+          <div class="video" data-video-id="eH0R10Ga4Hs"></div>
+        </div>
+      </li>
+      {% if LANG.startswith('en-') %}
+        <li class="feature">
+          <a href="https://hacks.mozilla.org/?p=28808" rel="external">
```

Comment:
[According to the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1159932#c15), this URL will be live on 6/2.